Names-values.Rmd

```diff
@@ -301,7 +301,7 @@ knitr::include_graphics("diagrams/name-value/d-modify-r.png", dpi = 300)
 ### Character vectors
 \index{string pool}
 
-The final place that R uses references is in character vectors. In the previous chapter, we drew character vectors like this:
+The final place where R uses references is in character vectors. In the previous section, we drew vectors like this:
```

Comment:
I misunderstood this because the "previous chapter" has now become the "next chapter". I thought this whole sentence was bizarre and I thought that it might refer to the vectors drawn in the previous section. Hence my edits.

Reading the "4. Vectors" chapter, I realized that the sentence initially made sense and simply was not working anymore because the place of "3. Names and values" must have changed at some point, since it is an additional chapter. So this section needs editing, but not the way I did...